docs/getting_started.rst

```diff
@@ -87,11 +87,11 @@ The most common targets are:
 .. code-block:: none
 
   $ npm run build       # runs the full build for both debug and compiled mode
-  $ npm run build:debug # runs the debug build only (requires Python)
+  $ npm run build:debug # runs the debug build only
   $ npm run test        # runs the unit tests
   $ npm run test:debug  # runs the unit tests with a configuration more suited to debugging
   $ npm run apidoc      # generates api documentation
-  $ npm run guide       # generates this documentation (requires Python and Sphinx)
+  $ npm run guide       # generates this documentation
```

Comment:
Would it be worth mentioning Sphinx in the deps list above (around line 17), something like "Python Sphinx (optional, needed only to generate the documentation)"?

Reply:
Yep, added. It's discussed in the section on how to build the RTD docs, but nice to have a full list of potential dependencies in one place.